pkg/playback/http_playback_test.go

```diff
@@ -0,0 +1,493 @@
+package playback
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/assert"
+	"github.com/stripe/stripe-go"
```

Comment:
So there's two disadvantages to this test:

1. It introduces a dependency on stripe-go, which has the potential to cause trouble -- the stripe-go interface might change and break this, we have to worry about keeping it updated, etc. Also stripe-go is a pretty big package.
2. Those `.bin` files are kind of big and introduce some indirection in understanding the test cases.

I think it was worthwhile to write this test the way you have, as proof that vcr works with actual stripe and a useful feedback loop, but I think we should rewrite the test in so that the fixtures are small enough to be defined inline, and are not specific to the stripe API. That is, keep `TestGetFromSimpleWebsite`, but don't hit `https://gobyexample.com` -- always hit the fixture server -- but have the fixture server's response be defined as a string inline (just like, "foo" or something) instead of loading it from a file -- remove the other test cases.

Reply:
Tests are updated as of 5ef6701718627002aabba37e4848137df95523ea